entries/pkid.go

```diff
@@ -95,3 +127,74 @@ func bulkDeletePkidEntry(entries []*lib.StateChangeEntry, db *bun.DB, operationT
 
 	return nil
 }
+
+func PkidBatchOperation(entries []*lib.StateChangeEntry, db *bun.DB, params *lib.DeSoParams) error {
+	// We check before we call this function that there is at least one operation type.
+	// We also ensure before this that all entries have the same operation type.
+	glog.Infof("PkidBatchOperation: Putting %d entries into the database", len(entries))
```

Comment:
Lets remove this before merging, or if it's necessary for debugging in a deployed environment, just throw a FIXME above it to remove.